server/app/middlewares/websocket_backend.rb

```diff
@@ -335,25 +377,33 @@ def on_pong(client, delay)
     end
 
     node = HostNode.find_by(node_id: client[:id])
+
     if node
-      if node.connected?
-        node.set(last_seen_at: Time.now.utc)
-      else
-        logger.warn "Close connection of disconnected node #{node.name || node.node_id}"
-        close_client(client)
+      connected_node = HostNode.where(id: node.id, connected_at: client[:connected_at])
+        .find_one_and_update({:$set => {last_seen_at: Time.now.utc}})
```

Comment:
Oh, sorry.. comment is in a wrong place.. just look for `{new: true}` , we need to change that with latest mongoid.